tests/RedisTest.php

```diff
@@ -1049,7 +1049,12 @@ public function testlPos()
         $this->assertEquals([0, 1], $this->redis->lPos('key', 'val1', ['count' => 2]));
         $this->assertEquals([0], $this->redis->lPos('key', 'val1', ['count' => 2, 'maxlen' => 1]));
         $this->assertEquals([], $this->redis->lPos('key', 'val2', ['count' => 1]));
-        $this->assertEquals(-1, $this->redis->lPos('key', 'val2'));
+
+        foreach ([[false, false], [true, NULL], [false, false]] as $optpack) {
```

Comment:
Why do you use `false, false` two times?

Reply:
Good point, `[[true, NULL], [false, false]]` makes more sense, I will update.